apps/frontend/components/Escrow/ReleaseFunds.tsx

```diff
@@ -36,42 +34,16 @@ const ReleaseFunds = ({ invoice, balance }: ReleaseFundsProp) => {
 
   const { address, currentMilestone, amounts, token } = invoice;
 
-  const waitForRelease = usePollSubgraph({
-    label: 'waiting for funds to be released',
-    fetchHelper: () => getInvoice(chainId, address),
-    checkResult: (updatedInvoice) => updatedInvoice.released > invoice.released,
-  });
-
   const onSuccess = async () => {
-    await waitForRelease();
     toast.success({ title: 'Funds released successfully' });
   };
 
   const { writeAsync: releaseFunds, isLoading } = useRelease({
-    invoice,
-    onSuccess,
+    invoice: { address },
+    onTxSuccess: onSuccess,
+    toast,
   });
 
-  // const pollSubgraph = async () => {
```

Comment:
lets remove these extra comments

Reply:
@dan13ram this is already removed. I think this is the before state